spec/models/item_spec.rb

```diff
@@ -24,4 +28,28 @@
     end
   end
 
+  describe '.valid_items' do
+    it 'returns all items that have not expired' do
+      expect(Item.valid_items).to include item
+    end
+
+    it 'does not return expired items' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.